tripper/backends/ontopy.py

```diff
@@ -10,7 +10,7 @@
 
 from ontopy.ontology import Ontology, _unabbreviate, get_ontology
 
-from tripper.triplestore import Literal
+from tripper import Literal
```

Comment:
```suggestion
from tripper.literal import Literal
```
Again, why not import directly from the module, where it exists, instead of the base package name? If the module is part of the public API, it's the more "accurate" method.